src/util/throttle.ts

```diff
@@ -12,7 +12,9 @@ export const throttle = <F extends (...args: any[]) => void>(fn: F, ms: number =
     if (timer) return;
     timer = setTimeout(() => {
       timer = 0;
-      fn.apply(null, lastArgs!);
+      if (lastArgs) {
+        fn.apply(null, lastArgs);
+      }
```

Comment:
Revert this change, it has no effect IMO. Or provide a unit test which fails if this `if (...)` condition is missing.